Browsers (#2090)
OfficeDev/office-js-docs-pr: docs/concepts/browsers-used-by-office-web-add-ins.md

|Windows 10 ver.&nbsp;>=&nbsp;1903 | Microsoft 365 ver.&nbsp;>=&nbsp;16.0.13127.20082<sup>1</sup>| No |Microsoft Edge<sup>2, 3</sup> with original WebView (EdgeHTML)|
|Windows 8.1<br>Windows 10| Microsoft 365 ver.&nbsp;>=&nbsp;16.0.13127.20082<sup>1</sup>| Yes|  SEE NOTE 4 BELOW.|

<sup>1</sup> See the [update history page](/officeupdates/update-history-office365-proplus-by-date) and how to [find your Office client version and update channel](https://support.office.com/article/What-version-of-Office-am-I-using-932788b8-a3ce-44bf-bb09-e334518b8b19) for more details.

<sup>2</sup> When Microsoft Edge is being used, the Windows 10 Narrator (sometimes called a "screen reader") reads the `<title>` tag in the page that opens in the task pane. When Internet Explorer 11 is being used, the Narrator reads the title bar of the task pane, which comes from the `<DisplayName>` value in the add-in's manifest.

<sup>3</sup> If your add-in includes the `Runtimes` element in the manifest, it uses Internet Explorer 11 regardless of the Windows or Microsoft 365 version. For more information, see [Runtimes](../reference/manifest/runtimes.md).

> <sup>4</sup> The browser that is used for this combination of versions depends on the channel of the Microsoft 365 subscription. If the user's subscription is on the [Beta channel](https://insider.office.com/join/windows) (formerly Insiders Fast channel), then Office uses Microsoft Edge with WebView2 (Chromium-based). For any other channel, Office uses Microsoft Edge with the original WebView (EdgeHTML). Support for WebView2 in other channels is expected in early 2021.

Reviewer: ```suggestion
<sup>4</sup> The browser that is used for this combination of versions depends on the update channel of the Microsoft 365 subscription. If the user is on the [Beta channel](https://insider.office.com/join/windows) (formerly Insiders Fast channel), then Office uses Microsoft Edge with WebView2 (Chromium-based). For any other channel, Office uses Microsoft Edge with the original WebView (EdgeHTML). Support for WebView2 in other channels is expected in early 2021.
```

The following table shows which browser is used for the various platforms and operating systems.

|**OS**|**Office version**|**Edge WebView2 (Chromium-based) installed**|**Browser**|
|:-----|:-----|:-----|:-----|:-----|:-----|:-----|

> |any|Office on the web|N/A|The browser in which Office is opened.|
> |Mac|any|N/A|Safari|
> |iOS|any|N/A|Safari|
> |Android|any|N/A|Chrome|

Reviewer: Should spell out N/A (see [style guide](https://docs.microsoft.com/en-us/style-guide/scannable-content/tables#content))
```suggestion
|any|Office on the web|Not applicable|The browser in which Office is opened.|
|Mac|any|Not applicable|Safari|
|iOS|any|Not applicable|Safari|
|Android|any|Not applicable|Chrome|
```

|Windows 7, 8.1, 10 | non-subscription Office 2013 or later|Doesn't matter|Internet Explorer 11|
|Windows 7 | Microsoft 365| Doesn't matter | Internet Explorer 11|

> |Windows 8.1,<br>Windows 10 ver.&nbsp;<&nbsp;1903| Microsoft 365 | No| Internet Explorer 11|

Reviewer: Do you need the `&nbsp;` encoding?
```suggestion
|Windows 8.1,<br>Windows 10 ver. < 1903| Microsoft 365 | No| Internet Explorer 11|
```

Author: Yes. To be consistent with other cells where it is needed to prevent bad line breaks and in case this cell's contents lengthen it will be needed to prevent a bad line break.

|Windows 10 ver.&nbsp;>=&nbsp;1903 | Microsoft 365 ver.&nbsp;<&nbsp;16.0.11629<sup>1</sup>| Doesn't matter|Internet Explorer 11|
|Windows 10 ver.&nbsp;>=&nbsp;1903 | Microsoft 365 ver.&nbsp;>=&nbsp;16.0.11629&nbsp;_AND_&nbsp;<&nbsp;16.0.13127.20082<sup>1</sup>| Doesn't matter|Microsoft Edge<sup>2, 3</sup> with original WebView (EdgeHTML)|
|Windows 10 ver.&nbsp;>=&nbsp;1903 | Microsoft 365 ver.&nbsp;>=&nbsp;16.0.13127.20082<sup>1</sup>| No |Microsoft Edge<sup>2, 3</sup> with original WebView (EdgeHTML)|

> |Windows 8.1<br>Windows 10| Microsoft 365 ver.&nbsp;>=&nbsp;16.0.13127.20082<sup>1</sup>| Yes|  SEE NOTE 4 BELOW.|

Reviewer: ```suggestion
|Windows 8.1<br>Windows 10| Microsoft 365 ver.&nbsp;>=&nbsp;16.0.13127.20082<sup>1</sup>| Yes|  Microsoft Edge<sup>4</sup>|
```

Author: I prefer to force the reader to read the note to get any information at all.

Reviewer: Ok, updated suggestion

<sup>1</sup> See the [update history page](/officeupdates/update-history-office365-proplus-by-date) and how to [find your Office client version and update channel](https://support.office.com/article/What-version-of-Office-am-I-using-932788b8-a3ce-44bf-bb09-e334518b8b19) for more details.

<sup>2</sup> When Microsoft Edge is being used, the Windows 10 Narrator (sometimes called a "screen reader") reads the `<title>` tag in the page that opens in the task pane. When Internet Explorer 11 is being used, the Narrator reads the title bar of the task pane, which comes from the `<DisplayName>` value in the add-in's manifest.

<sup>3</sup> If your add-in includes the `Runtimes` element in the manifest, it uses Internet Explorer 11 regardless of the Windows or Microsoft 365 version. For more information, see [Runtimes](../reference/manifest/runtimes.md).

> <sup>4</sup> The browser that is used for this combination of versions depends on the channel of the Microsoft 365 subscription. If the subscription uses the [Beta channel](https://insider.office.com/join/windows) (formerly Insiders Fast channel), then Office will use Microsoft Edge with WebView2 (Chromium-based). For any other channel, Office will use Microsoft Edge with original WebView (EdgeHTML). Support for WebView2 in other channels is expected in early 2021.

Reviewer: ```suggestion
<sup>4</sup> The browser that is used for this combination of versions depends on the update channel of the Microsoft 365 subscription. If you're on the [Beta channel](https://insider.office.com/join/windows) (formerly Insiders Fast channel), then Office uses Microsoft Edge with WebView2 (Chromium-based). For any other channel, Office uses Microsoft Edge with the original WebView (EdgeHTML). Support for WebView2 in other channels is expected in early 2021.
```

Author: I've made most of these changes, and a modified version of one, but this article doesn't use "you" to refer to the user and I want to be consistent.

> 
> 

Reviewer: ```suggestion

```
